pkg/cmd/pipelinerun/log_reader.go

```diff
@@ -50,7 +50,7 @@ func (lr *LogReader) Read() (<-chan Log, <-chan error, error) {
 	tkn := lr.Clients.Tekton
 	pr, err := tkn.TektonV1alpha1().PipelineRuns(lr.Ns).Get(lr.Run, metav1.GetOptions{})
 	if err != nil {
-		return nil, nil, fmt.Errorf("%s : %s", msgPRNotFoundErr, err)
+		return nil, nil, fmt.Errorf(err.Error())
```

Comment:
nit: for the future, we should think of using `golang.org/x/xerrors` to wrap errors :angel: